daprdocs/content/en/reference/components-reference/supported-pubsub/setup-alicloud-rocketmq.md

```diff
@@ -0,0 +1,111 @@
+---
+type: docs
+title: "Alibaba Cloud RocketMQ"
```

Comment:
Reword to remove all references to Alibaba. This is a generic component. As per type `type: pubsub.rocketmq`

Reply:
This also means renaming this file to remove alibaba.